src/components/BToggle/BToggle.md

```diff
@@ -0,0 +1,17 @@
+```js
+
+const toggleValue1 = false
```

Comment:
We should show the output for components in the documentation somehow

Reply:
This file just feeds the live style guide, which will render the components.  We decided to move the examples into a separate file so we wouldn't clutter the .vue file.  Let me know if I am misunderstanding what you were saying.

Reply:
I think she means we need to somehow show in the docs what the component spits back out